site/content/docs/5.2/migration.md

```diff
@@ -33,6 +33,11 @@ If you need more details about the changes, please refer to the [v5.2.2 release]
 
 - <span class="badge bg-warning">Warning</span> All forms examples have been modified to add a `.mt-2` to all submit buttons in order to always have 30px between the last form control and the button. Please reflect this modification into your websites.
 
+### Contents
+
+- **Tables**
+  - When using row selection (checkboxes), use `.has-checkbox` on the table to get correct spacing on the first column.
```

Comment:
Suggestion:

```suggestion
  - Reintroduction of `.has-checkbox` from v4 to display correctly the row selection in the first column of tables. 
```

Reply:
I also add a warning because, if this is a migration guide from 5.1 to 5.2, adding has-checkbox class is needed...